FBAllocationTracker/NSObject+FBAllocationTracker.h

```diff
@@ -31,3 +31,12 @@
 @end
 
 #endif // _INTERNAL_FBAT_ENABLED
```

Comment:
If I put this code inside _INTERNAL_FBAT_ENABLED check, then the library code won't compile in release configuration.